libs/shared/ofrep-core/package.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "@openfeature/ofrep-core",
+  "version": "0.0.1",
```

Can we start with an experimental version here maybe?
```suggestion
  "version": "0.0.1-experimental",
```